lib/ReactViews/DragDropFile.jsx

```diff
@@ -27,13 +27,27 @@ const DragDropFile = createReactClass({
             .then(addedCatalogItems => {
                 if (addedCatalogItems.length > 0) {
                     this.props.viewState.myDataIsUploadView = false;
-                    this.props.viewState.viewCatalogMember(addedCatalogItems[0]);
+                    if(this.props.viewState.explorerPanelIsVisible) {
+                        this.props.viewState.viewCatalogMember(addedCatalogItems[0]);
+                    } else {
+                        this.notifyUpload(addedCatalogItems);
+                    }
                 }
             });
 
         this.props.viewState.isDraggingDroppingFile = false;
     },
 
+    notifyUpload(addedCatalogItems) {
+        // if explorer panel is not open, we show a notification
+        // slide in right, wait 3 + 2 seconds (plus transition time), then slide out right
+        this.props.viewState.recentlyUploadedFiles = addedCatalogItems.map(item => item.name);
+        setTimeout(
+            ()=> {
+                this.props.viewState.recentlyUploadedFiles = [];
+            },5000);
```

Comment:
Would it be hard to keep the notification open if the mouse cursor is over it? That'd be a nice touch I think.

Reply:
Moved this timeout into `DragDropNotificaiton` to handle hover.